### DISCUSSION TITLE
Remove `wire_expression2_debug`

### DISCUSSION THREAD

**Discussion by vurvdev** [wiremod/wire#2798] [Suggestions]
I don't think it has been useful for years.

Currently trying to declutter registerType (at least index 5 is pointless, only used in `guess_type` which shouldn't exist in the first place, and this debugger).

All it does is hook all e2 functions and operators, and invokes the type checker on all arguments passed to the functions.

Our test suite should really cover any need for this now..

**Comment by vurvdev** [comment]
Since this is only developer facing and I'm the current head maintainer of E2, I'm going to go ahead and remove it regardless

**Comment by vurvdev** [comment]
Also going to remove GLON since it's so incredibly old and hasn't been mentioned on the discord or github in the capacity of it actually being used.

(GLON was the only thing using the stupid `guess_type` function)

**Comment by Denneisk** [comment]
Fixed in #2799
